src/query/functions/src/scalars/geometry.rs

```diff
@@ -53,6 +75,44 @@ pub fn register(registry: &mut FunctionRegistry) {
         })
     );
 
+    registry.register_passthrough_nullable_1_arg::<StringType, GeometryType, _, _>(
+        "st_geometryfromwkt",
+        |_, _| FunctionDomain::Full,
```

Comment:
`parse_to_ewkb` may failed, the function domain should be `FunctionDomain::MayThrow`